### PR TITLE
[FEATURE] Software versions

### PR DESCRIPTION
**Pull request by Zethson**
I'm missing `msstats_plfq.R` for the software versions. How do we fetch the version number for this?

### CODE REVIEW

**Comment by Zethson** [comment]
Ah whops. Just found it. It's in the bin folder :man_facepalming:

File: bin/scrape_software_versions.py [hunk 1]
```diff
@@ -7,14 +7,10 @@
 regexes = {
     'nf-core/proteomicslfq': ['v_pipeline.txt', r"(\S+)"],
     'Nextflow': ['v_nextflow.txt', r"(\S+)"],
-    'FastQC': ['v_fastqc.txt', r"FastQC v(\S+)"],
```
Comment (Zethson) [inline bin/scrape_software_versions.py]: Have to add our stuff here as well. I'll do it.

**Comment by Zethson** [comment]
@jpfeuffer ready

**Comment by jpfeuffer** [comment]
I assume something fails and the infinity bug kicks in.

**Comment by Zethson** [comment]
Huh. I didn't even wait for the CI, since I didn't expect this to fail.
I'll look into it tomorrow.

**Comment by Leon-Bichmann** [comment]
While reviewing the template update yesterday I saw there was a change in the scrape software versions py script. Maybe that is causing this fail now?

**Comment by Zethson** [comment]
I already know why it's failing. Two reasons:

1. The software versions process was not updated with the latest template, but the python script was (I fixed that one)

2. The OpenMS processes send an error signal, which have to be redirected in a proper way, or else the nextflow script will simply fail (I fixed that one with @drpatelh pointer)

3. Current issue: For some damn reason the regex ain't picking up the file now. I'm trying stuff out.

**Comment by Zethson** [comment]
```

nf-core/proteomicslfq	v1.0dev
--
Nextflow	v19.10.0
ThermorawfileParser	v1.2.0
FileConverter	v2.5.0
DecoyDatabase	v2.5.0
MSGFPlusAdapter	v2.5.0
MSGFPlus	vv2017.07.21
CometAdapter	v2.5.0
Comet	v2016.01 rev. 3
PeptideIndexer	v2.5.0
PSMFeatureExtractor	v2.5.0
PercolatorAdapter	v2.5.0
Percolator	v3.04.0
IDFilter	v2.5.0
IDScoreSwitcher	v2.5.0
FalseDiscoveryRate	v2.5.0
IDPosteriorErrorProbability	v2.5.0
ProteomicsLFQ	v2.5.0
MSstats	v1.0dev
```

**Comment by Zethson** [comment]
@jpfeuffer Finally, it should be ready. Sorry for the too early request :)

**Comment by jpfeuffer** [comment]
Looks good! It's a bit weird that MSGF gets a second "v" but I guess NF just adds a "v" everywhere?

**Comment by Zethson** [comment]
It's the scrape_software_versions.py script that does that:

results[k] = "v{}".format(match.group(1))

**Comment by Zethson** [comment]
I guess with a smarter regex we can take care of this. I may do that in another PR later.

**Pull request merged**